Database/DATA_DICTIONNARY.md

```diff
@@ -3,24 +3,34 @@ Topic discussed in [related issue](https://github.com/Oppskrift/oppskrift_api/is
 
 ## Recipe
 
-Property | Type | Description | Exemples | Comments | 
------------- | ------------- | ------------- | ------------- | ------------- |
-name | String | Explicit | `Avocado toast` | ✔️ |
-author | String | Explicit | `Martin` | ✔️ |
-description | String | Explicit | `Some dummy description...` | ✔️ |
-image | String | Explicit | `dummyUrl` | Could it be an array of images ? |
-cookTime | Duration in ISO format | Explicit | `PT40M` | ✔️ |
-recipeCategory | String | Explicit | `dessert` | Could a recipe have several categories ? |
-recipeCuisine | String | type of cuisine (French, Mexican, …) | `Japanese` | Could a recipe have several cuisines ? |
-recipeIngredient | Array | List of ingredients | `['salt', 'pepper', 'cucumber']` | schema.org says it's a single ingredient but it's a list of ingredients |
-recipeInstructions | Array | Explicit | `['wash salad', 'cut salad']` | ✔️ |
-recipeYield | Number | quantity produced (number of servings, number of people) | `6` | Does it needs a related variable unit ? |
-suitableForDiet | ??? | Explicit | --- | Need an example |
-estimatedCost | Number | Explicit | --- | Not sure it is needed because mainly relative to regions/countries, if we keep, what about currency ? |
-prepTime | Number | Explicit | `3600` | Need to select a unit |
-**Properties from Recipe** | ??? | Specific properties related to a Recipe | --- | --- |
+Property | Type | Description | Examples | Is calculated (not persisted) | Comments |
+------------ | ------------- | ------------- | ------------- | ------------- | ----- |
+name | `String` | Explicit | `Avocado toast` | |  |
+author | `String` | Explicit | `Martin` |  |  |
+description | `String` | Explicit | `Some dummy description...` |  | Optional |
+image | `String or Array<String>` | Explicit | `dummyUrl` |  | Single url or array of urls, Optional |
+cookTime | Duration in `ISO` format | Explicit | `PT40M` | Optional |
+recipeCategory | `String or Array<String>` | Explicit | `dessert` |  |  |
+recipeCuisine | `String` | type of cuisine (French, Mexican, …) | `Japanese` |  | Optional |
+recipeIngredients | `Array<String or Object>` | List of ingredients | `['salt', 'pepper', ' 1 cucumber', '200ml milk']` | Is it raw ingredient or is it computed one with qty and unit ? |
+recipeInstructions | `Array<String>` | Explicit | `['wash salad', 'cut salad']` |  |  |
+recipeYield | `String` | quantity produced (number of servings, number of people) | `1 loaf`, `300g of salad`, `12 servings` or `4 personnes` |  |  |
+suitableForDiet | `Array<String>` | Explicit | `['vegan', 'gluten free']` |  | Optional |
+prepTime | Duration in `ISO` format | Explicit | `3600` |  |  |
+waitTime | Duration in `ISO` format | Explicit | `3600` |  | Optional |
```

Comment:
Unfortunately the waitTime has not been accepted as a valuable addition to schema.org. I guess it's not relevant to keep it

Reply:
But, regarding your comment maybe it's a good idea to keep it, let's approve it !